tendrl/monitoring_integration/graphite/__init__.py

```diff
@@ -259,6 +274,14 @@ def get_central_store_data(self, objects):
                         copy.deepcopy(resource_detail))
                 node_list = self.get_resource_keys(cluster_key, "nodes")
                 for node in node_list:
+                    node_deleted_key = os.path.join(cluster_key, "nodes", node,
```

Comment:
@shtripat @rishubhjain  NS.tendrl.objects.NodeContext.deleted does not exist. What are you guys trying to do here?

https://github.com/Tendrl/commons/blob/master/tendrl/commons/objects/node_context/__init__.py

Reply:
@r0h4n Yes, I will have to update the code, the deleted flag is supposed to be in Peers (clusters/{cluster_id}/Peers/{peer_id}). I have just discussed it with @shtripat , Will make the necessary changes.

Reply:
Discussion on https://github.com/Tendrl/gluster-integration/pull/440#discussion_r142849226 will be affecting this PR, so this development will be put on hold untill I get an ack from @shtripat

Reply:
As per https://github.com/Tendrl/gluster-integration/pull/440#discussion_r142850578 from Atin, the setting of deleted flag for peer while peer_disconnect event is not correct and should not be done. So effectively, there is not going to be a deleted flag set for cluster peers in the event of peer_disconnect.

Later if we support peer_detach event in tendrl, this flag might be used. So I feel NodeContext.status is the only field which should be used to decide here.